### PR TITLE
Added Table of Contents

### PR DESCRIPTION
**Pull request by woshiicyrus**
This is to keep the README file neat. It will ensure that user able to navigate to each section easily.

### CODE REVIEW

**Comment by ClobberXD** [comment]
Is the `Command-Line options` section worth linking to? It just says `Use --help`

File: README.md [hunk 1]
```diff
@@ -16,6 +16,18 @@ If you downloaded the Minetest Engine source code in which this file is
...
 project too. See its README.txt for more information.
 
+Table of Contents
+------------------
+
+1. [Further Documentation](#further-documentation)
+2. [Default Controls](#default-controls)
+3. [Paths](#paths)
+4. [Configuration File](#configuration-file)
+5. [Command-Line Options](#command-line-options)
```
Comment (ClobberXD) [inline README.md]: `-line`, not `-Line`

**Comment by woshiicyrus** [comment]
Thank you for the correction. By the way, since the Command-line option is a header, that is why I thought it is necessary. If you do not think that it is necessary, I can remove that line though.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by ClobberXD** [comment]
Just voicing my thoughts... I'm unsure about removing it. while it's just one single line, it still isn't trivial, and I feel that it should be easy to find.

**Comment by woshiicyrus** [comment]
I suggested this because when I first navigated through the file, it was quite confusing. So, I hope that in the future no one would get confused. I hope that you would merge this. Thank you

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by ClobberXD** [comment]
> I hope that you would merge this. Thank you

And I hope you aren't referring to me here. I was just providing my inputs; I'm not a core-dev ;)

**Comment by woshiicyrus** [comment]
Alright. Thanks for the feedback !

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by srifqi** [comment]
Thank you for you contribution!

Have you read [this guide](https://github.com/minetest/minetest/blob/master/CONTRIBUTING.md)? It seems like, you don't create another branch for this PR.

**Comment by srifqi** [comment]
Thank you for your contribution!

Have you read [this guide](https://github.com/minetest/minetest/blob/master/CONTRIBUTING.md)? It seems like you didn't create another branch for this PR.

**Comment by paramat** [comment]
Trivial PR, anyone support? I'm neutral.

**Comment by ClobberXD** [comment]
My only minor nit was the inclusion of the `Command-line options` section in the TOC, which just contains one line: `Use --help`. But I think it should stay, because ideally the actual command-line options should be listed out in that section for a quick reference. I guess that could be added separately sometime later. Otherwise, this PR is :+1:

**Comment by pauloue** [comment]
I don't think it's necessary. Better clean the file up and make it smaller.

**Comment by Ezhh** [comment]
There's nothing wrong with having a content section, and generally it's pretty useful.

When deciding on which content links to have, you go by heading level and shouldn't deviate from that; if there are sections people feel aren't worthy of a heading, then it's suggesting maybe they should be incorporated somewhere else and not have a dedicated section at that level.

**Comment by paramat** [comment]
:+1:

**Pull request merged**